### ISSUE TITLE
TileSet collision layer not importing

### ISSUE THREAD

**Issue by JSchrepp** [Kiamo2/YATI#5]
**Describe the bug**
While the reference doc is unclear as to naming convention, the code suggests that setting custom property `collision_layer` on a tileset will result in the parsing of that property value to be set as a collision layer(s). I have a tileset with this custom property set, shown below in-app and in the `.tsx` file:

![image](https://user-images.githubusercontent.com/15899212/228390923-da2cd183-0b7f-46ef-877c-90eb2949235d.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<tileset version="1.10" tiledversion="1.10.0" name="testset" tilewidth="32" tileheight="32" tilecount="8" columns="2">
 <properties>
  <property name="collision_layer" value="4,5"/>
 </properties>
```

Looking at the [code here](https://github.com/Kiamo2/YATI/blob/7a95b988fefd9a0a854ae5873398d11d57331942/GDScript/addons/YATI/TilesetCreator.gd#L567) it seems this is the expected behavior. However, the resulting `.tscn` file doesn't end up with the correct collision layer. Instead, it has the default collision layer 1 and metadata called "collision layer" set to the value I specified.

```
[sub_resource type="TileSet" id="TileSet_ypy1m"]
tile_shape = 1
tile_layout = 5
tile_size = Vector2i(32, 16)
physics_layer_0/collision_layer = 1
sources/0 = SubResource("TileSetAtlasSource_ltbpe")
metadata/collision_layer = "4,5"
```

Clearly my value is being found but it is not managing to trigger as a well-known value, instead being added to generic metadata.

**To Reproduce**
1. Create tileset with a collision layer and a tilemap using that tileset
2. import the map using plugin

**Expected behavior**
The appropriate physics layers should be set as outlined

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 1.1.2
 - Godot Version 4.0.1.stable.official [cacf49999]
 - Tiled Version 1.10.0

**Comment by Kiamo2** [comment]
Hi, you have to name the property "collision_layer_0" (or "collision_layer_1" and so forth).
The reason is that Godot has physics layers and the collision polygons are children of those.
So "collision_layer_0" sets "collision_layer" on physics layer 0.
I didn't have a better idea to be both short (only one property) and flexible (be able to target all physics layers).
BUT... since version 1.2.0 it's now possible to only name it "collision_layer".
In that case physics layer 0 is targeted.
You are using version 1.1.2 according to your issue description.
Hope that helps?

**Comment by Kiamo2** [comment]
Hi, you have to name the property "collision_layer_0" (or "collision_layer_1" and so forth).
The reason is that Godot has physics layers and the collision polygons are children of those.
So "collision_layer_0" sets "collision_layer" on physics layer 0.
I didn't have a better idea to be both short (only one property) and flexible (be able to target all physics layers).
BUT... since version 1.2.0 it's now actually possible to only name it "collision_layer" :-)
In that case physics layer 0 is targeted.
You are using version 1.1.2 according to your issue description.

Hope that helps?

PS nevertheless I detected another bug which needs urgent updating, I will asap release v 1.2.1

**Comment by Kiamo2** [comment]
That's right, that exactly was the bug I fixed in 1.2.1
